tests/alerts/test_session_opened_sensitive_user.py

```diff
@@ -126,16 +126,6 @@ class TestSessionOpenedUser(AlertTestSuite):
         )
```

Comment:
Do we need to remove lines like https://github.com/mozilla/MozDef/blob/master/tests/alerts/test_geomodel.py#L14  from all of our alert unit tests?

Reply:
Not yet, not until we actually split the indexes.